internal/healthchecks/network_check.go

```diff
@@ -20,9 +20,43 @@ import (
 )
 
 var (
-	// API urls
-	loggingAPIUrl    = "https://logging.googleapis.com/$discovery/rest"
-	monitoringAPIUrl = "https://monitoring.googleapis.com/$discovery/rest"
+	requests = []struct {
+		name             string
+		url              string
+		successMessage   string
+		healthCheckError HealthCheckError
+	}{
+		{
+			name:             "Logging API",
+			url:              "https://logging.googleapis.com/$discovery/rest",
+			successMessage:   "Request to the Logging API was successful.",
+			healthCheckError: LogApiConnErr,
+		},
+		{
+			name:             "Monitoring API",
+			url:              "https://monitoring.googleapis.com/$discovery/rest",
+			successMessage:   "Request to the Monitoring API was successful.",
+			healthCheckError: MonApiConnErr,
+		},
+		{
+			name:             "Packages API",
+			url:              "https://packages.cloud.google.com",
+			successMessage:   "Request to packages.cloud.google.com was successful.",
+			healthCheckError: PacApiConnErr,
+		},
+		{
+			name:             "dl.google.com",
+			url:              "https://dl.google.com",
+			successMessage:   "Request to dl.google.com was successful.",
+			healthCheckError: DLApiConnErr,
+		},
+		{
+			name:             "GCE Metadata Server",
+			url:              "http://metadata.google.internal",
+			successMessage:   "Request to the GCE Metadata server was successful.",
+			healthCheckError: MetaApiConnErr,
+		},
+	}
```

Comment:
Would it make sense (maybe as a later refactor after this PR) to merge this struct with the one from `error.go`? Everything looks like it's one-to-one so it might make more sense to just have everything related to a check be all in one place...

Reply:
Lets refactor in another another PR.